docs/data/data-grid/row-updates/row-updates.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Data Grid - Row updates
+---
+
+# Data grid - Row updates
```

Comment:
This page will require a lot more content in v6 with #4927 and with all the lazy-loading related topics that @DanailH is handling.

It could even require a dedicated page folder at some point with a page for each approach.
But for now, we can improve this one.

Reply:
Does it make sense to merge this page with "row definition", to keep consistency with https://mui.com/x/react-data-grid/column-definition/#providing-content? In essence, we're talking here about different approachs to feed the grid with rows. One of the approachs also allows to update rows (`updateRows`), but it could also be understood as replacing one row with another (updated) row, so "providing rows via API".

Reply:
@joserodolfofreitas @samuelsycamore I'm interested in your opinion here

I agree that this section could be on the same page as the row update sections

![image](https://user-images.githubusercontent.com/3309670/177279079-95995173-6882-422b-a43d-46f5a5cfc0c8.png)


But having the "Row identifier" section in the same page as well will start to make the page very long.

Reply:
Would it make sense to split that page into 2, one for `Row updates` and one for `Row loading`? The `Row loading` one will have the infinite + lazy loading and anything else related to loading rows on the fly.

Reply:
Yeah, I agree it makes sense to move the lazy loading to its own page.
We could probably use one more depth level on the side bar to hold everything about "feeding data", but I agree it makes sense to separate `Row updates` from `Row Loading` (and merge `Row definitions` with `Row updates`).

But maybe we could rename this menus?

What about

- Data binding
- Lazy loading

?

Reply:
It can work but calling the page that is about loading rows "Lazy loading" won't be ideal as my idea was to have lazy and infinite loading there.

Reply:
then:

- Data binding
- Lazy and infinite loading

?

Reply:
@joserodolfofreitas I would add items to the **left sidebar**. Keeping "lazy loading" inside https://deploy-preview-5195--material-ui-x.netlify.app/x/react-data-grid/row-updates/ will make the page very long. Note that we need to consider that "Row updates" and "Row definition" will be merged.

Reply:
@DanailH @m4theushw @joserodolfofreitas 
Did we reach an agreement on this topic? Or should I create a separate issue for discussion to unlock this PR?